### PR TITLE
mrc-4706 Skeleton CLI

### PR DESCRIPTION
**Pull request by EmmaLRussell**
This skeleton kotlin cli app will be the basis of the admin CLI for the new shiny server (twinkle). The CLI will be used to add new apps to the server. 

The branch just includes the boilerplate gradle set up, and has placeholder commands for "add" to add an app and "list" to list existing apps (might be useful to have that for sanity testing before we get the web portal up and running). 

A gha runs tests and linting with detekt.

Future branches will provide docker container for running the app, and sqllite schema and migrations to actually keep the list of apps. 

Each app will have a name, a git repo url and a list admin users (github ids). 

To test, run e.g.
`./gradlew :run --args="add newApp https://github.com/fakeOrg/fakeRepo user1 user2"`
or
`./gradlew :run --args="list"`

Uses Clikt: https://ajalt.github.io/clikt/

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.jar
Comment (richfitz) [inline gradle/wrapper/gradle-wrapper.jar:1]: committed by accident, or is this really something that gets committed?

File: gradlew.bat [hunk 1]
```diff
@@ -0,0 +1,89 @@
+@rem
+@rem Copyright 2015 the original author or authors.
```
Comment (richfitz) [inline gradlew.bat:2]: old! is this really what gets written out?

File: gradle/wrapper/gradle-wrapper.jar
Comment (EmmaLRussell) [reply gradle/wrapper/gradle-wrapper.jar:1]: Yep, gradle comes along for the ride I'm afraid.

File: gradlew.bat (hunk 1 above)
Comment (EmmaLRussell) [reply gradlew.bat:2]: if it ain't broke..I guess!

**Review by absternator — APPROVED** [review]
Cool little library.. good base LGTM

**Pull request merged**